js/app/widgets/eknWebview.js

```diff
@@ -176,9 +177,31 @@ const EknWebview = new Knowledge.Class({
                 if (scheme === 'license') {
                     let license = GLib.uri_unescape_string(uri.replace('license://', ''), null);
                     uri = Endless.get_license_file(license).get_uri();
+
+                    let license_view = new EosKnowledgePrivate.WebViewDialog({
+                      'transient-for' : this.get_toplevel(),
+                      'width-request' : 640,
+                      'height-request': 720,
+                      'home-uri'      : uri,
+                    });
+
+                    license_view.webview.connect('decide-policy', (view, decision, type) => {
+                      let uri = decision.request.uri;
```

Comment:
You could use `GLib.uri_parse_scheme` to decide instead of simply looking at the prefix.